site/docs/runtimes/deno.mdx

```diff
@@ -52,3 +52,35 @@ const sql = query.compile()
 
 console.log(sql.sql)
 ```
+
+Deno works well with [PostgresJS](https://github.com/porsager/postgres), to that end you can use the Kysely dialect and the driver:
```

Comment:
I wasn't sure if I should remove the explanation above this one so I left it there.